lib.php

```diff
@@ -18,6 +18,11 @@
 
 define('XAPI_REPORT_ID_ERROR', 0);
 define('XAPI_REPORT_ID_HISTORIC', 1);
+
```

Comment:
remove this extra line or add a comment here, pls.

Reply:
I have added a comment here to indicate this section.